cmd/provider/daemon.go

```diff
@@ -11,12 +11,12 @@ import (
 	datatransfer "github.com/filecoin-project/go-data-transfer/impl"
 	dtnetwork "github.com/filecoin-project/go-data-transfer/network"
 	gstransport "github.com/filecoin-project/go-data-transfer/transport/graphsync"
-	"github.com/filecoin-project/indexer-reference-provider/config"
-	"github.com/filecoin-project/indexer-reference-provider/engine"
-	"github.com/filecoin-project/indexer-reference-provider/internal/cardatatransfer"
-	"github.com/filecoin-project/indexer-reference-provider/internal/suppliers"
-	adminserver "github.com/filecoin-project/indexer-reference-provider/server/admin/http"
-	p2pserver "github.com/filecoin-project/indexer-reference-provider/server/provider/libp2p"
+	"github.com/filecoin-project/indexer-reference-provider/lib/config"
+	"github.com/filecoin-project/indexer-reference-provider/lib/engine"
+	"github.com/filecoin-project/indexer-reference-provider/lib/cardatatransfer"
+	"github.com/filecoin-project/indexer-reference-provider/lib/suppliers"
```

Comment:
Maybe rename this `supplier` to follow the no plurals package naming convention

Reply:
Done!